### PR TITLE
Removing padding from main Container

### PR DESCRIPTION
**Pull request by cwg83**
Some views need to be the full width, so it makes more sense to have padding: 0 on the main Container component and have each view handle it's own spacing

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ export default function App() {
     <ThemeProvider theme={theme}>
       <CssBaseline />
       {<TopBar setUser={handleSetUser} changeView={changeView} profileData={profileData}/>}
-      <Container id='main-app-container' maxWidth={'sm'}>
```
Comment (nsfoster87) [inline client/src/components/App.jsx:107]: Any reason not to put this stying inside a css file?

**Review by nsfoster87 — APPROVED** [review]
Any reason not to move this styling inside a css file? I like the thought process here, how do we feel about adding some CSS Resets to the top of our CSS File?

**Comment by cwg83** [comment]
Yeah we might want to decide on how we want to handle stuff. I think there are pros and cons for each method, and we may want to use a combination of them depending on our use cases. For something like this where we are removing some default style that MUI adds that we don't believe we need, I think you're correct that adding a class or ID and then adding the style to the CSS file makes more sense.

If we need to change a component's style based on state or something, inline styling makes a bit more sense. There are a few other interesting points to consider that I found here:
https://www.engineyard.com/blog/inline-styles-yes-or-no/

**Comment by cwg83** [comment]
Oh one more thing to keep in mind is that we can't use the Theme styles in our CSS file, but again that wouldn't apply here.

**Pull request merged**